python/lsst/pipe/base/pipeline_graph/_pipeline_graph.py

```diff
@@ -712,6 +712,7 @@ def add_task(
             label = task_class._DefaultName
         if config is None:
             config = task_class.ConfigClass()
+        _LOG.debug("Task: %s %s", label, task_class)
```

Comment:
I think something more verbose would be more helpful when disambiguating debug logs. Maybe something like `Adding task with label: %s and class: %s to the pipeline graph`

Reply:
The current ticket produces output that looks like this:
```
lsst.pipe.base.pipeline_graph DEBUG: Task: photometricMatchVisit <class 'lsst.analysis.tools.tasks.photometricCatalogMatch.PhotometricCatalogMatchVisitTask'>
```

Your suggestion would produce output that looks like this:
```
lsst.pipe.base.pipeline_graph DEBUG: Adding task with label: photometricMatchVisit and class: <class 'lsst.analysis.tools.tasks.photometricCatalogMatch.PhotometricCatalogMatchVisitTask'> to the pipeline graph
```

Is the double "class" what you had in mind?

Reply:
Having discussed in-person, modifying to:
```
Adding task: %s %s to the pipeline graph
```